src/main.rs

```diff
@@ -65,8 +65,20 @@ pub struct Flags {
     #[structopt(long, help = "Enable jwt middleware")]
     jwt: Option<Option<Flipper>>,
 
+    #[structopt(long, help = "Enable live reload in development")]
+    livereload: Option<Option<Flipper>>,
```

Comment:
Does `Option<None>` here mean something different than just `None`? I realize this is beyond this PR but generally this is an anti-pattern, since there isn't any logical difference. (Or maybe this is just some `structopt` weirdness? I haven't used it.)

Reply:
This is structopt weirdness to allow for `--livereload` (`Some(None)`, equiv to `--livereload=on`), `--livereload=on` (`Some(Some("on"))`), and `--livereload=off` (`Some(Some("off"))`).